src/components/card/_macro-options.md

```diff
@@ -18,3 +12,21 @@
 | largeSrc       | string | false    | Path to the retina version of the image                                                                              |
 | alt            | string | false    | The HTML `alt` tag to explain the appearance and function of the image. Not required if the image is only decorative |
 | placeholderURL | string | false    | Optional base path for placeholder image                                                                             |
+
+## Title
+
+| Name         | Type   | Required | Description                                                                   |
+| ------------ | ------ | -------- | ----------------------------------------------------------------------------- |
+| text         | string | true     | The text for the title of the card heading                                    |
```

Comment:
This wording is confusing maybe change the description to "The text for the card title"

Reply:
done